### PR TITLE
Show Multiple nodes per safe

### PR DESCRIPTION
**Pull request by mjadach-iv**
image

### CODE REVIEW

File: src/store/slices/stakingHub/actionsAsync.ts [hunk 1]
```diff
@@ -527,13 +534,17 @@ export const createAsyncReducer = (builder: ActionReducerMapBuilder<typeof initi
     }
   });
   builder.addCase(getNodeDataThunk.fulfilled, (state, action) => {
-    if(action.payload.length > 0) {
-      state.nodes.push(action.payload[0]);
-    }
-    state.nodes[0].isFetching = false;
+    console.log('action', action)
```
Comment (esterlus) [inline src/store/slices/stakingHub/actionsAsync.ts:537]: intentional log?

File: src/store/slices/stakingHub/actionsAsync.ts [hunk 2]
```diff
@@ -527,13 +534,17 @@ export const createAsyncReducer = (builder: ActionReducerMapBuilder<typeof initi
...
+    // if(action.payload.length > 0) {
+    //   state.nodes[]
+      
+      
+    // }
+    // state.nodes[0].isFetching = false;
   });
   builder.addCase(getNodeDataThunk.rejected, (state, action) => {
-    state.nodes[0].isFetching = false;
+    console.log('action', action)
```
Comment (esterlus) [inline src/store/slices/stakingHub/actionsAsync.ts:546]: intentional log?

**Review by esterlus — APPROVED** [review]

File: src/store/slices/stakingHub/actionsAsync.ts (hunk 1 above)
Comment (mjadach-iv) [reply src/store/slices/stakingHub/actionsAsync.ts]: yep

File: src/store/slices/stakingHub/actionsAsync.ts (hunk 2 above)
Comment (mjadach-iv) [reply src/store/slices/stakingHub/actionsAsync.ts]: yep

**Review by Jaguaras — APPROVED** [review]
Lgtm

**Pull request merged**